openwisp-config/files/sbin/openwisp-update-config.lua

```diff
@@ -32,7 +32,7 @@ local test_root_dir = working_dir .. '/update-test'
 local remote_dir = openwisp_dir .. '/remote'
 local remote_config_dir = remote_dir .. '/etc/config'
 local stored_dir = openwisp_dir .. '/stored'
-local stored_config_dir = openwisp_dir .. '/etc/config'
+local stored_config_dir = stored_dir .. '/etc/config'
```

Comment:
This is exactly the code I had in my PR back then:

https://github.com/TDT-AG/openwisp-config/commit/3e1bf40b1aec7109e94c043dde40e017fa02bc94#diff-30f1cf0abf884291b3f1d13aa28690d35fde2464f42b74205369ea6417715abeR30

So you must have modified it when adding the test to my PR :grin:

Reply:
My bad for not catching that :-1: 
@devkapilbansal definitely please pay more attention when doing this kind of changes.